resources/views/admin/modules/index.blade.php

```diff
@@ -0,0 +1,61 @@
+@extends('admin.app')
+@section('title', "Modules Manager")
+@section('actions')
+  <li>
+    <a href="{{ route('admin.modules.create') }}">
+      <i class="ti-plus"></i>
+      Add New</a>
+  </li>
+@endsection
+@section('content')
+  <div class="card border-blue-bottom">
+    <div class="content">
+      <div class="row">
+        <div class="col-lg-12">
+          <table class="table table-bordered table-primary">
+            <thead>
+              <th>Module</th>
+              <th>Status</th>
+              <th>Actions</th>
+            </thead>
+            <tbody>
+            @forelse($modules as $module)
+              <tr>
+                <td>
+                  {{$module->name}}
+                  @if($module->is_new == 1)
```

Comment:
Don't need this, have a separate section for the scanned modules that aren't installed

Reply:
Alright!